src/main/webapp/js/adminActivityLog.js

```diff
@@ -41,13 +41,11 @@ function submitLocalTimeAjaxRequest(time, googleId, role, entry){
         success : function(data) {
             setTimeout(function(){
                 if (!data.isError) {   	
-                	$(link).parent().html(originalTime + "<mark>" + "<br>" + data.getLogLocalTime()) + "</mark>";
+                	$(link).parent().html(originalTime + "<mark>" + "<br>" + data.logLocalTime) + "</mark>";
                 } else {
                 	$(localTimeDisplay).html("Loading error, please retry");      	
                 }
-            	               
-                $("#statusMessage").html(data.getStatusForAjax());
-
+                $("#statusMessage").html(data.statusForAjax);
```

Comment:
Although not related to this pr, does this line do anything? statusForAjax is always undefined for me.